### DISCUSSION TITLE
Is there a way to copy/paste the custom extension content

### DISCUSSION THREAD

**Discussion by nsvetozarevic** [ueberdosis/tiptap#4263] [General Questions]
Hey,

I created a custom node extension, with interactive node views (with Vue). It works perfectly, but one of the requirements that my client has is that users could select the extension alongside other editors' content and copy/paste it.
I couldn't find anything in the docs, do you have at least some guidance in which direction I should go?
I can see that it works perfectly for e.g. built-in marks (Bold, Italic)

Thanks!

**Comment by millievn** [comment]
no solution until now
